src/test/java/steps/NewCheckingAccountSteps.java

```diff
@@ -0,0 +1,73 @@
+package steps;
+
+import io.cucumber.java.BeforeAll;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.junit.jupiter.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import pages.LoginToBank;
+import utils.Driver;
+
+import java.util.concurrent.TimeUnit;
+
+public class NewCheckingAccountSteps {
+
+    WebDriver driver = Driver.getDriver();
+    private LoginToBank loginToBank = new LoginToBank(driver);
+
+    @BeforeAll
+    public static void setup() {
+
+        WebDriverManager.firefoxdriver().setup();
+    }
+
+    @Given("user is on {string}")
+    public void user_is_on(String string) {
+
+        driver.manage().window().maximize();
+        driver.get("https://dbank-qa.wedevx.co/bank/login");
```

Comment:
Since you're passing the url in your feature file, you can reuse the parameter instead of hardcoding the url